Test failure in org.csstudio.swt.rtplot

This seems to be a race condition.  If I increase the last wait time in the test from 300 to 500 it appears to pass reliably.

```
org.csstudio.swt.rtplot.util.UpdateThrottleTest.testUpdateThrottle

Failing for the past 1 build (Since Failed#80 )
Took 2.6 sec.
add description
Error Message

Expected: <4>
     but: was <3>

Stacktrace

java.lang.AssertionError: 
Expected: <4>
     but: was <3>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.csstudio.swt.rtplot.util.UpdateThrottleTest.testUpdateThrottle(UpdateThrottleTest.java:76)

Standard Output

-> Trigger..
<- Update!
-> Trigger..
-> Trigger..
-> Trigger..
<- Update!
<- Update!
<- Update!
```

--- a/applications/appunorganized/appunorganized-plugins/org.csstudio.swt.rtplot.test/src/org/csstudio/swt/rtplot/util/UpdateThrottleTest.java
+++ b/applications/appunorganized/appunorganized-plugins/org.csstudio.swt.rtplot.test/src/org/csstudio/swt/rtplot/util/UpdateThrottleTest.java
@@ -72,7 +72,7 @@ public class UpdateThrottleTest
         throttle.trigger();
         TimeUnit.MILLISECONDS.sleep(100);
         assertThat(updates.get(), equalTo(3));
-        TimeUnit.MILLISECONDS.sleep(300);
+        TimeUnit.MILLISECONDS.sleep(1000);
         assertThat(updates.get(), equalTo(4));
     }
 }

